oak_containers_launcher/src/server.rs

```diff
@@ -189,6 +195,27 @@ impl Launcher for LauncherServerImplementation {
     }
 }
 
+#[tonic::async_trait]
+impl HostlibKeyProvisioning for LauncherServerImplementation {
+    async fn get_key_provisioning_role(
+        &self,
+        _request: Request<()>,
+    ) -> Result<Response<GetKeyProvisioningRoleResponse>, tonic::Status> {
+        // TODO(#4442): Implement setting Hostlib Key Provisioning role via an input argument.
+        Ok(tonic::Response::new(GetKeyProvisioningRoleResponse {
+            role: KeyProvisioningRole::Leader.into(),
+        }))
+    }
+
+    async fn get_group_keys(
+        &self,
+        _request: Request<()>,
+    ) -> Result<Response<GetGroupKeysResponse>, tonic::Status> {
+        // TODO(#4442): Implement sending group keys to the orchestrator.
+        Ok(tonic::Response::new(GetGroupKeysResponse::default()))
```

Comment:
Why does this return Ok? Shouldn't it return Err?

Reply:
Done